aggregator/src/rlc.rs

```diff
@@ -0,0 +1,4 @@
+mod config;
```

Comment:
I suggest move `rlc` module inside the `aggregation` modules as it is a gadget used by aggregation circuit only.

Reply:
will fix after steven's integration test passes